lib/kubeclient/inflections.rb

```diff
@@ -0,0 +1,3 @@
+ActiveSupport::Inflector.inflections do |inflect|
+  inflect.irregular 'quota', 'quotas'
```

Comment:
Is this global? Any way to scope this just for this gem? Maybe @matthewd or @tenderlove know that.